server/src/main/java/com/google/fhir/proxy/ProxyConstants.java

```diff
@@ -20,8 +20,25 @@
 
 public class ProxyConstants {
 
+  public static final String CARE_TEAM_TAG_URL = "http://smartregister.org/fhir/care-team-tag";
```

Comment:
@ekigamba  You can keep the resource url here and append the rest of the server domain name from the environment variable PROXY_TO. 
This holds the URL of the HAPI server.

Reply:
Sorry, this is the Coding.system in which is usually a url such as `http://snomed.info/sct` and `https://acme.lab/resultcodes`. Do we expect these to change?

Reply:
We can bring this up for discussion or we can change all the `coding.system` s back to `http://smartregister.org` as defined in this issue https://github.com/opensrp/fhircore/issues/1596

Reply:
Yes, I understand. Thanks for clarifying, 
I was only concerned with the server domain name. @ekigamba 
Because it might get changed if we move to another deployment instance, I am unsure if I got it right.
I will review the issue you have referenced in a while.

Reply:
Cool, we can possibly make the whole `Code.system` configurable then in that case